lib/generators/logidze/install/templates/migration.rb.erb

```diff
@@ -189,6 +192,25 @@ class <%= @migration_class_name %> < ActiveRecord::Migration<%= ActiveRecord::VE
               RETURN NEW;
             END IF;
 
+            IF (
+              debounce_time IS NOT NULL AND
+              (version->>'ts')::bigint - (NEW.log_data#>'{h,-1,ts}')::text::bigint <= debounce_time
+            ) THEN
+              -- merge new version with the previous one
+              version := NEW.log_data#>'{h,-1}' || version;
```

Comment:
this will merge the `v`, `ts`, (meta) `m` and (responsible_id) `r` along with the changesets `c`. However, `||` can't do the deep merge for the changeset so we need to do it separate in the next line.

For meta `m`, we can do a deep merge but just like you said, I don't think it's neccesary